src/operators/windowCount.ts

```diff
@@ -25,29 +25,34 @@ class WindowCountOperator<T, R> implements Operator<T, R> {
 }
 
 class WindowCountSubscriber<T> extends Subscriber<T> {
-  private windows: { count: number, window: Subject<T> } [] = [];
+  private windows: { count: number, notified : boolean, window: Subject<T> } [] = [{ count: 0, notified : false, window : new Subject<T>() }];
   private count: number = 0;
   
   constructor(destination: Observer<T>, private windowSize: number, private startWindowEvery: number) {
-    super(destination); 
+    super(destination);
   }
   
   _next(value: T) {
     const count = (this.count += 1);
-    const startWindowEvery = this.startWindowEvery;
+    const startWindowEvery = (this.startWindowEvery > 0) ? this.startWindowEvery : this.windowSize;
```

Comment:
As mentioned in https://github.com/ReactiveX/RxJS/pull/271#discussion_r38808559, some function initialize default value to 0 and some sets it to null for `number` types. As same as previous changes only updates value but not default value since not sure if there's recommended guidelines.